sync-core/src/main/java/com/cloudant/sync/query/IndexUpdater.java

```diff
@@ -248,53 +249,52 @@ public Boolean call() {
         }
```

Comment:
the msg string above here should say String.format("%s. Only one array field per index allowed", msg)